demo/script/gen-icons.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+const fs = require("fs");
+const {
+  findIcons,
+  generateIconset,
+  genMDIIcons,
+} = require("../../gulp/tasks/gen-icons.js");
+
+const INFO_OUTLINE = "information-outline";
```

Comment:
Can you add a comment why this one has to be included?

Reply:
It doesn’t... I forgot to remove it

Reply:
removed